examples/mnist/Instructions.md

```diff
@@ -0,0 +1,25 @@
+<p align="center"><b>How to test example MNIST train/test programs</b><p align="center">
```

Comment:
Or simply `build` might be good.

There's already detailed explanation for source code of example programs.
https://github.com/tiny-dnn/tiny-dnn/blob/master/examples/mnist/readme.md

So what kind of instructions does `examwples/mnist/Instructions.md` tell?
How to checkout and build?